Core/include/Acts/Geometry/DetectorVolume.hpp

```diff
@@ -346,6 +361,9 @@ class DetectorVolume : public std::enable_shared_from_this<DetectorVolume> {
   /// Volume store (internal/external)
   ObjectStore<std::shared_ptr<DetectorVolume>> m_volumes;
 
+  /// BoundingBox
+  std::shared_ptr<BoundingBox> m_boundingBox;
```

Comment:
Maybe 
```suggestion
  std::shared_ptr<const BoundingBox> m_boundingBox;
```
if we don't intend to modify it after creation.

Reply:
Done.